shared/js/abp-preprocessed.es6.js

```diff
@@ -140,3 +147,41 @@ function getVersionParam () {
 
     return versionParam
 }
+
+function fetchSurrogateCode () {
+    let trackersSurrogateListEtag = settings.getSetting('trackersSurrogateList-etag') || ''
```

Comment:
Looks like since this is happening on bootup and depends on `settings.getSetting()` which can be laggy especially on Firefox, you'll want to definitely invoke the call to this function with `settings.ready().then()` promise, otherwise the stored etag is never going to be read.